### PR TITLE
Add query/locals for ruby syntax highlighting

### PR DESCRIPTION
**Pull request by TravonteD**

### CODE REVIEW

**Comment by kyazdani42** [comment]
also could you squash the commits into one please ?

File: queries/ruby/locals.scm [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+(destructured_parameter name: (identifier) @local.definition)
+(block_parameter name: (identifier) @local.definition)
+(keyword_parameter name: (identifier) @local.definition)
+
+(identifier) @local.reference
+
+(assignment left:(identifier) @local.definition)
+(left_assignment_list (identifier) @local.definition)
+(rest_assignment (identifier) @local.definition)
+(destructured_left_assignment (identifier) @local.definition)
```
Comment (vigoux) [inline queries/ruby/locals.scm:45]: One of the differences between atom queries and ours is that we completely remove the `local` part of the capture group. If it is possible to change this, we'll be ready to merge.

**Comment by vigoux** [comment]
One little comment though, we've been working on normalizing the highlights query, which might need some modification in the future.

**Comment by TravonteD** [comment]
I've updated the locals.scm and squashed the commits into one. Let me know if there is anything else.

**Comment by vigoux** [comment]
LGTM, thanks for the contribution, we will reach out to you when a norm is found !

**Pull request merged**